stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.11.0
+version: 11.11.1
```

Comment:
You bumped the `prometheus` chart and not the `unifi` chart, FYI!

Reply:
Bahahahaha last week was rough